.travis.yml

```diff
@@ -20,7 +20,6 @@ matrix:
     - php: 7.0
     - php: 7.1
     - php: 7.2
-    - php: 7.0
```

Comment:
> But isn’t it already included for this version just a few lines up?

@719media: If I'm reading this diff correctly, it combines two separate jobs (PHP 7.2 with server 3.6.2 and PHP 7.0 with server 3.0.15) with one (PHP 7.2 with server 3.0.15).

For no real reason in particular, we run tests with all older server versions under PHP 7.0.